pkg/platform/aws/template.go

```diff
@@ -104,6 +93,72 @@ module "aws-{{.Config.ClusterName}}" {
   {{- end }}
 }
 
+{{ range $index, $pool := .Config.WorkerPools }}
+module "worker-pool-{{ $index }}" {
+  source = "../lokomotive-kubernetes/aws/flatcar-linux/kubernetes/workers"
+
+  providers = {
+    aws      = aws.default
+  }
+
+  vpc_id                = module.aws-{{ $.Config.ClusterName }}.vpc_id
+  subnet_ids            = flatten([module.aws-{{ $.Config.ClusterName }}.subnet_ids])
+  security_groups       = module.aws-{{ $.Config.ClusterName }}.worker_security_groups
+  kubeconfig            = module.aws-{{ $.Config.ClusterName }}.kubeconfig
```

Comment:
Nit:

```suggestion
  vpc_id          = module.aws-{{ $.Config.ClusterName }}.vpc_id
  subnet_ids      = flatten([module.aws-{{ $.Config.ClusterName }}.subnet_ids])
  security_groups = module.aws-{{ $.Config.ClusterName }}.worker_security_groups
  kubeconfig      = module.aws-{{ $.Config.ClusterName }}.kubeconfig
```

Reply:
@iaguis I am indenting w.r.t `cluster_domain_suffix = "{{ $.Config.ClusterDomainSuffix }}"`

Reply:
alright, that's fine then.

Reply:
I also pointed it out here: https://github.com/kinvolk/lokomotive/pull/151#discussion_r393516800 :smile: I think we could put `cluster_domain_suffix` in separate "block" and then reduce the indentation for all other parameters.